dev/integration_tests/ios_platform_view_tests/ios/PlatformViewUITests/PlatformViewUITests.m

```diff
@@ -36,17 +36,31 @@ - (void)setUp {
   if ([appIcon waitForExistenceWithTimeout:kStandardTimeOut]) {
     NSLog(@"Deleting previously installed app.");
 
-    // Make icons wiggle
-    [appIcon pressForDuration:3];
+    if (!appIcon.isHittable) {
```

Comment:
Instead of sleeping, use the `wait` APIs
https://github.com/flutter/flutter/blob/7428ab66b4e79b695ffc03a6e15a2d18b019aa7f/dev/integration_tests/ios_host_app/FlutterUITests/FlutterUITests.m#L120-L122